bots/BotLib/hero_primal_beast.lua

```diff
@@ -39,57 +39,51 @@ else
     nTalentBuildList    = J.Skill.GetTalentBuild(tTalentTreeList[2])
 end
 
-local sUtility = {"item_pipe", "item_lotus_orb", "item_crimson_guard"}
-local nUtility = sUtility[RandomInt(1, #sUtility)]
-
 local sRoleItemsBuyList = {}
 
 sRoleItemsBuyList['pos_2'] = {
     "item_tango",
-    "item_quelling_blade",
-    "item_double_branches",
+	"item_enchanted_mango",
+	"item_enchanted_mango",
+	"item_enchanted_mango",
+	"item_clarity",
+	"item_clarity",
+	"item_flask",
+	"item_faerie_fire",
+	"item_quelling_blade",
 
-    "item_bracer",
+    "item_tranquil_boots",
```

Comment:
Do you think it's necessary to change the boots and remove bracer? Both items are quite strong, and tranquil boots have worse winrate in human matches🤔

Reply:
primal is also broken like io, he can't do lasthit which causes less gold per minute. tranquil is cheap and prevents him from buying tango and healing salve which costs gold. bracer is also waste of money because broken heroes act stupid, so they cannot benefit from it. he always dies until he became real TANK. he always have worst networth like io, we must sacrefise some items until lasthit problem got fixed. (he also cannot hit jungle creeps most of the time, thats why i gave him radiance otherwise he just looks at them)